aktuelt/models.py

```diff
@@ -99,7 +100,7 @@ def schedule(self):
 
     api_fields = [
         APIField("intro"),
-        APIField("body"),
+        APIField("body", serializer=NewsBodySerializer()),
```

Comment:
There is also the option of going with https://docs.wagtail.org/en/stable/topics/streamfield.html which seems to allow a few more advanced options. Mainly I think it could be useful if we wanted to transition to JSON array output rather than html and leave more of the body rendering logic to frontend (and remove the need for extending/hacking into wagtail frontend related code).